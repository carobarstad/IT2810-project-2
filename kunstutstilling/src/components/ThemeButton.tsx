import React, {useContext} from 'react'
import ThemeContext from './ThemeContext'

export default function ThemeButton() {
    const {dark, toggle} = useContext(ThemeContext)
    
    return (
        <button className="SettingsButton" onClick={toggle}>    
            Toggle theme        
        </button>
    )
}
