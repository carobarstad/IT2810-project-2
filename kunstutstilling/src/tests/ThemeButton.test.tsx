import React from 'react';
import ThemeButton from '../components/ThemeButton';
const ReactTestRenderer = require('react-test-renderer');

describe('Button', () => {
  it('matches snapshot', () => {
    const tree = ReactTestRenderer.create(
      <ThemeButton />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});