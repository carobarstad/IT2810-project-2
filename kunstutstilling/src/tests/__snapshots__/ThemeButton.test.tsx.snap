// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
<button
  className="SettingsButton"
  onClick={[Function]}
>
  Endre tema
</button>
`;
